optuna/storages/_rdb/models.py

```diff
@@ -36,7 +36,7 @@
     _Column = mapped_column
 except ImportError:
     # TODO(Shinichi): Remove this after dropping support for SQLAlchemy<2.0.
-    from sqlalchemy import Column as _Column  # type: ignore[assignment, no-redef]
+    from sqlalchemy import Column as _Column
```

Comment:
@Alnusjaponica 
Could you simply ignore this line as this line will be removed sooner or later anyways?

```suggestion
    from sqlalchemy import Column as _Column  # type: ignore[assignment]
```